README.md

```diff
@@ -1,83 +1,112 @@
+Notes:
+* create a list of dishes with prices - a hash sounds best
 
-In code review we'll be hoping to see:
+```
+require './lib/takeaway'
+takeaway = Takeaway.new
+takeaway.read_menu # should return menu hash
+```
+### User story 2
+> As a customer  
+> So that I can order the meal I want  
+> I would like to be able to select some number of several available dishes
+
+Need to be able to place order for several dishes  
+Objects: customer, order  
+Actions: place_order  
+Notes:
+* customer to be able to order multiples of each dish
+* confirmation sentence after each item
+* store order in a hash, with name of menu item as key and number of menu items as value.  Then refer to dishes hash when calculating total
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+```
+require './lib/takeaway'
+takeaway = Takeaway.new
+takeaway.place_order("pizza", 3) # should add 3 pizzas to an order
+```
+### User story 3
+> As a customer  
+> So that I can verify that my order is correct  
+> I would like to check that the total I have been given matches the sum of the various dishes in my order
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this at this moment.
+Need to check that total is correct  
+Objects: customer, order, total  
+Actions: total
 
-Notes on Test Coverage
-------------------
+```
+require './lib/takeaway'
+takeaway = Takeaway.new
+takeaway.place_order("pizza", 3)
+takeaway.place_order("burger")
+takeaway.total == "Total: £24.96" # should return true
+```
+### User story 4
+> As a customer  
+> So that I am reassured that my order will be delivered on time  
+> I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
+
+Need to receive a text advising that the order will be delivered before one hour from now. Need to link to current time! Need to utilise Twilio API (text message API).  
+Objects: customer, order  
+Actions: checkout(generates text)  
+Notes:
+* register for Twilio - done
+* update Gemfile to include twilio-ruby gem - done
+* use own mobile number - make this private & don't push to GitHub - done!
+* also need to keep Twilio API key private - check how to do this on GitHub - done
+* tests shouldn't send texts, so use mocking or stubs here
 
-You can see your [test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) when you run your tests.
+```
+require './lib/takeaway'
+takeaway = Takeaway.new
+takeaway.place_order("pizza", 3)
+takeaway.place_order("burger")
+takeaway.checkout # should generate text message confirming order, and return confirmation message.
+```
```

Comment:
Really nice README! It was great to read your plan for each of the user stories, as well as an example usage of the app!